UIKitDz/ViewController.swift

```diff
@@ -7,6 +7,162 @@ import UIKit
 class ViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
-        view.backgroundColor = .red
+
+        view.backgroundColor = .white
+        addNumbers()
+        addGuess()
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        introduceYourself()
+    }
+
+    // MARK: Метод для ввода имени пользователя
+
+    func introduceYourself() {
+        let name = UIAlertController(title: "Представьтесь", message: "Введите ваше имя", preferredStyle: .alert)
```

Comment:
лучше давать имя чтобы ты сама потом разобралась 
```suggestion
        let alertViewController = UIAlertController(title: "Представьтесь", message: "Введите ваше имя", preferredStyle: .alert)
```